components/brave_wallet/renderer/js_ethereum_provider.h

```diff
@@ -50,12 +53,10 @@ class JSEthereumProvider final : public gin::Wrappable<JSEthereumProvider>,
                     base::Value result) override;
 
  private:
-  explicit JSEthereumProvider(content::RenderFrame* render_frame);
-  ~JSEthereumProvider() override;
-
   class MetaMask final : public gin::Wrappable<MetaMask> {
    public:
-    static gin::WrapperInfo kWrapperInfo;
+    static constexpr gin::WrapperInfo kWrapperInfo = {{gin::kEmbedderNativeGin},
+                                                      gin::kMojoWatcher};
```

Comment:
Shouldn't it be smth like kMetamask?

Reply:
cc: @cdesouza-chromium

Reply:
Fixed in 4c6de9f2bdd. Thanks for noticing that @cypt4